### ISSUE TITLE
Failing test test-admin-deploy-karg.sh

### ISSUE THREAD

**Issue by Mathnerd314** [ostreedev/ostree#342]
Quoting from [the test](https://github.com/ostreedev/ostree/blob/master/tests/test-admin-deploy-karg.sh#L46):
>  Here we're asserting that the *host* system has a root= kernel
 argument.  I think it's unlikely that anyone doesn't have one, but
 if this is not true for you, please file a bug!

```
$ cat /proc/cmdline 
initrd=\efi\nixos\zijhk6j55mszzcafhyqslfhmacxi4wlf-initrd-initrd.efi systemConfig=/nix/store/vh5ksfxaf62095s627lcl7zjhjpd04gl-nixos-system-nixos-16.09undefined init=/nix/store/vh5ksfxaf62095s627lcl7zjhjpd04gl-nixos-system-nixos-16.09undefined/init
```

A similar error occurs in ``tests/test-admin-instutil-set-kargs.sh``.

**Comment by cgwalters** [comment]
Likely the best here is to re-parse the cmdline in python or something and validate they have the same content.

**Comment by smcv** [comment]
I believe this was fixed in a94530111a31ff3784af293a6cd4.

**Comment by cgwalters** [comment]
:white_check_mark:
